### PR TITLE
update onnx, add tiny_yolo_v2

### PR DESCRIPTION
**Pull request by joddiy**

### CODE REVIEW

File: docs-site/website/versioned_docs/version-2.0.0/onnx.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
-id: version-2.0.0-onnx
```
Comment (FeynmanDNA) [inline docs-site/website/versioned_docs/version-2.0.0/onnx.md]: @joddiy hi for versioned_docs you have to keep the predix

File: docs-site/website/versioned_docs/version-2.0.0/onnx.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
+id: onnx
 title: ONNX
-original_id: onnx
```
Comment (FeynmanDNA) [inline docs-site/website/versioned_docs/version-2.0.0/onnx.md]: @joddiy same here, for versioned_docs, you need to keep the `original_id`

File: docs-site/website/versioned_docs/version-2.0.0/onnx.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 ---
-id: version-2.0.0-onnx
+id: onnx
```
Comment (nudles) [inline docs-site/website/versioned_docs/version-2.0.0/onnx.md]: I think we do not neet to duplicate the documentation into versioned_docs/ as this documentation will be available in v2.1. The code is now in singa:dev.

File: docs-site/website/versioned_docs/version-2.0.0/onnx.md (hunk 1 above)
Comment (joddiy) [reply docs-site/website/versioned_docs/version-2.0.0/onnx.md]: Hi, @FeynmanDNA , sorry for late, just fix it.

File: docs-site/website/versioned_docs/version-2.0.0/onnx.md (hunk 2 above)
Comment (joddiy) [reply docs-site/website/versioned_docs/version-2.0.0/onnx.md]: Hi, @FeynmanDNA , sorry for late, just fix it.

File: docs-site/website/versioned_docs/version-2.0.0/onnx.md (hunk 3 above)
Comment (joddiy) [reply docs-site/website/versioned_docs/version-2.0.0/onnx.md]: @FeynmanDNA do you have any idea about this?

File: docs-site/website/versioned_docs/version-2.0.0/onnx.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
-id: onnx
+id: version-2.0.0-onnx
```
Comment (FeynmanDNA) [inline docs-site/website/versioned_docs/version-2.0.0/onnx.md]: @joddiy hi, if i understand prof wang's suggestion, we can ignore the update for `versioned_docs` for v2.0.0, as this update only applies to v2.0+

Comment (nudles) [reply docs-site/website/versioned_docs/version-2.0.0/onnx.md]: yes. version 2.0.0 is already fixed. all new files or updates will be in v2.1

**Pull request merged**